src/audio/audio.hpp

```diff
@@ -105,6 +105,19 @@ class Audio
 	 * @see Seek
 	 */
 	virtual std::uint64_t Position() const = 0;
+
+	/**
+	 * This Audio's length.
+	 * @return Audio's length, in microseconds.
+	 */
+	virtual std::uint64_t TotalLength() const = 0;
```

Comment:
I figured since Position() used a uint64_t, it would make sense to use the same for this